OrbitGl/App.cpp

```diff
@@ -383,6 +385,10 @@ void OrbitApp::AddSamplingReport(std::shared_ptr<SamplingProfiler> sampling_prof
     sampling_reports_callback_(callstack_data_view, report);
   }
 
+  // clear old sampling report
```

Comment:
why is this needed? maybe add a comment explaining?

Reply:
The main reason for this is clearing CallStackDataView (the field `CallStackDataView* callstack_data_view_;` in the SamplingReport). Should I write more detailed comment here?